core/src/main/java/nl/nn/adapterframework/batch/StreamTransformerPipe.java

```diff
@@ -219,9 +208,7 @@ public void registerChild(IRecordHandlerManager manager) throws Exception {
 		ConfigurationWarnings.add(this, log, "configuration using element 'child' is deprecated. Please use element 'manager'", ConfigurationWarnings.DEPRECATION_SUPPRESS_KEY, getAdapter());
 		registerManager(manager);
 	}
-	/**
-	 * Register a uniquely named manager.
-	 */
+	@IbisDoc({"1", "a uniquely named manager"})
```

Comment:
I do not agree with "1" for the order here. We already put some @IbisDoc annotations for sort orders in an earlier pull request. There we use 10, 20, 30 etc, such that we have the option to insert new methods anywhere in the existing sort order.

Reply:
done